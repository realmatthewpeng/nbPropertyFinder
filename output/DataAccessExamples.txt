Total number of lines found: 19

teams_all[teams_all.name== 'Boise St.'].sort()
games_all
fts = fts[(fts.name_1.notnull()) & (fts.name_2.notnull())]
features_total = features_total[cols]
fts['value'] = value
features_total = pd.concat([features_1, features_2])
features_total.drop(['team1','team2','year','name_1','name_2'], inplace=True, axis=1)
t_B_fts = teams[(teams.name==team_b) & (teams.year == year)]
fts.drop(['name_1','name_2','year'], inplace=True, axis=1)
fs
t_A_fts = teams[(teams.name==team_a) & (teams.year == year)]
temp = t[0]
wt = kag_results.ix[kag_results.season==season, ['wteam', 'lteam']]
teams_combined = pd.merge(pd.merge(kag_seeds[kag_seeds.season==season].sort(columns='team'),
teams[teams.year==year],
teams.ix[teams.name==team[0],'name'] = team[1]
wt['result'] = [checker(t) for t in wt.values]
t[0] = t[1]
res = results[results.res.notnull()]

Incorrect Lines:

features_total = features_total[cols]
temp = t[0]
wt = kag_results.ix[kag_results.season==season, ['wteam', 'lteam']]
teams_combined = pd.merge(pd.merge(kag_seeds[kag_seeds.season==season].sort(columns='team'),
teams[teams.year==year],
teams.ix[teams.name==team[0],'name'] = team[1]
wt['result'] = [checker(t) for t in wt.values]
t[0] = t[1]
res = results[results.res.notnull()]

Precision = 0.5263157894736842
Recall = 0.9090909090909091

results[results.res.notnull()]
