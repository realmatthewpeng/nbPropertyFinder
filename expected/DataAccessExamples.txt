teams_all[teams_all.name== 'Boise St.'].sort()
games_all
features_total = pd.concat([features_1, features_2])
features_total.drop(['team1','team2','year','name_1','name_2'], inplace=True, axis=1)
results[results.res.notnull()]
t_B_fts = teams[(teams.name==team_b) & (teams.year == year)]
t_A_fts = teams[(teams.name==team_a) & (teams.year == year)]
fts.drop(['name_1','name_2','year'], inplace=True, axis=1)
fts['value'] = value
fts = fts[(fts.name_1.notnull()) & (fts.name_2.notnull())]
fs